src/components/Steps/Dataset/Prepare.js

```diff
@@ -40,10 +39,10 @@ class DatasetPrepare extends Component {
     }
 
     render() {
-        const { t, list } = this.props
+        const { t, wordlist } = this.props
```

Comment:
What is `t`?

Reply:
`t` is shorthand for `react-i18next` library's string translating. Let's mention this in the docs.